core/domain/state_domain.py

```diff
@@ -772,65 +772,76 @@ def validate(self):
                 self.needs_update)
 
 
-class ContentTranslations(object):
+class WrittenTranslations(object):
     """Value object representing a content translations which stores
     translated contents of all state contents (like hints, feedback etc.) in
     different languages linked through their content_id.
     """
 
-    def __init__(self, content_translations):
-        """Initializes a ContentTranslations domain object."""
-        self.content_translations = content_translations
+    def __init__(self, translations_mapping):
+        """Initializes a WrittenTranslations domain object."""
+        self.translations_mapping = translations_mapping
 
     def to_dict(self):
-        """Returns a dict representing this ContentTranslations domain object.
+        """Returns a dict representing this WrittenTranslations domain object.
 
         Returns:
-            dict. A dict, mapping all fields of ContentTranslations instance.
+            dict. A dict, mapping content_id to a dict containing language_code
+            as a key and written translation dict as a value.
         """
-        content_translations_dict = {}
+        written_translations_dict = {}
```

Comment:
I thought I wrote in a [previous comment](https://github.com/oppia/oppia/pull/6175#discussion_r255292884) that the "dict form" should match the "object form" as far as possible? I.e. the dict should have a 'translations_mapping' key...